app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -17,6 +17,7 @@ import org.oppia.android.domain.profile.ProfileManagementController
 import org.oppia.android.util.data.DataProviders.Companion.toLiveData
 import javax.inject.Inject
 
+/** Argument key for Profile Deletion dialog in [ProfileEditFragment]. */
```

Comment:
Nit
`/** Argument key for profile deletion dialog in [ProfileEditFragment]. */`

Reply:
done